wiremock/mappings/no_state_about_to_submit.json

```diff
@@ -0,0 +1,19 @@
+{
+  "request": {
+    "method": "POST",
+    "urlPath": "/case_type/no_state/about_to_submit"
```

Comment:
we should not have case type names and callback names on the path. To allow these json to be fully reusable in other case types and callbacks.
Also try to reuse as much as possible json files already existing rather than introducing new ones.
You could rename this urlPath to:
`callback_returning_only_data`